spec/Section 5 -- Validation.md

```diff
@@ -1669,9 +1768,10 @@ query variableIsNotDefined {
 ${atOtherHomes} is not defined by the operation.
 
 Fragments complicate this rule. Any fragment transitively included by an
-operation has access to the variables defined by that operation. Fragments can
-appear within multiple operations and therefore variable usages must correspond
-to variable definitions in all of those operations.
+operation has access to the variables defined by that operation and defined on
+the fragment. Fragments can appear within multiple operations and therefore
```

Comment:
This is kind of fine as-is but reads a little ambiguously if the read is not careful (as if the variable has to be defined in both places). Adding `also those` seems to clarify this:

```suggestion
operation has access to the variables defined by that operation and also those defined on
the fragment. Fragments can appear within multiple operations and therefore
```